aws/resource_aws_pinpoint_app.go

```diff
@@ -128,6 +128,12 @@ func resourceAwsPinpointApp() *schema.Resource {
 					},
 				},
 			},
+			"arn": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
The ARN of the PinPoint Application cannot be set by the operator, so `Optional: true` should be removed. 👍 

```suggestion
```

Reply:
Removed.